apis/beacon/states/committee.yaml

```diff
@@ -7,32 +7,26 @@ get:
   parameters:
     - name: state_id
       in: path
-      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/StateId'
+      $ref: "../../../beacon-node-oapi.yaml#/components/parameters/StateId"
```

Comment:
makes it harder to see changes if we're changing quotes when not needed..

Reply:
Right, auto-formater messed the commit, let me revert

Reply:
Reverted all formatting changes :+1: